static/usr/libexec/core/get-arg

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+# Usage:
+#   get-arg param-name
+#
+# get-arg will look for kernel parameter "param-name" or "param_name"
+# and return 0 if found, 1 if not found. If the parameter as a value,
+# e.g. "param-name=the-value", then the value "the-value" will be
+# printed.
+
+# For more information on how to parse kernel parameters, see function
+# `next_arg` in
+# https://github.com/torvalds/linux/blob/master/lib/cmdline.c
+
+set -eu
+
+if [ "$#" -ne 1 ]; then
+    echo "Expected kernel parameter name as argument" 1>&2
+    exit 1
+fi
+
+looking_for="$(echo "${1}" | sed 's/_/-/g')"
+
+if [ "${SYSTEMD_PROC_CMDLINE:+set}" = set ]; then
+    # Using same debug variable as systemd for testing
+   cmdline="${SYSTEMD_PROC_CMDLINE}"
+else
+   cmdline=$(cat /proc/cmdline)
+fi
+
+set --
+
+# We cannot use ANSI-C quoting (e.g. $'\n') in busybox-initramfs
+# dash's printf builtin, so we make sure does not handle \x
+whitespaces="$(/usr/bin/printf '\t\n\v\f\r \xA0')"
```

Comment:
Note that in the end octal can be used, see https://github.com/snapcore/core-base/pull/129

Reply:
Noted, thanks. 

My suggestion is that we back-port the official fix so that this diff will fall away on final merge.